packages/react-query/src/bin/utils/copy.ts

```diff
@@ -10,7 +10,7 @@ export function copy(version: number) {
     return false
   }
 
-  files.forEach((file) => {
+  for (const file of files) {
```

Comment:
ref: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-for-each.md

This may not have a significant impact on our project, but I agree with the intent and would like to follow it.